templates/utils/template_funcs.go

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package commands
+// Package utils contains the utility function for template commands.
+
+package utils
```

Comment:
Can we name this "common" rather than "utils"? Seems a bit more descriptive

Reply:
Done.